Frontend/app/src/App.tsx

```diff
@@ -1,23 +1,82 @@
-import React from 'react';
-import { useState, useEffect } from "react";
+import React, { useState, useEffect, ChangeEvent } from 'react';
+import axios from 'axios';
 import './App.css';
+import FileList from './components/FileList';
+import FileUpload from './components/FileUpload';
+import Message from './components/Message';
+import DeleteAllFiles from './components/DeleteAllFiles';
 
 function App() {
-
-  const [currentTime, setCurrentTime] = useState(0);
+  const [selectedFile, setSelectedFile] = useState<File | undefined>(undefined);
+  const [message, setMessage] = useState('');
+  const [files, setFiles] = useState<string[]>([]);
 
   useEffect(() => {
-    fetch(`${process.env.REACT_APP_BACKEND_URL}/time`).then(res => res.json()).then(data => {
-      setCurrentTime(data.time);
-    });
+    fetchFiles();
   }, []);
 
+  const fetchFiles = async () => {
+    try {
```

Comment:
recommend using .env variables for the url

Reply:
Good idea if we want to keep the endpoints private. The `process.env.REACT_APP_BACKEND_URL` was not working which is why I hardcoded it as a temp solution but can fix it